src/Microsoft.AspNetCore.Mvc.Core/RedirectToActionResult.cs

```diff
@@ -53,6 +73,11 @@ public class RedirectToActionResult : ActionResult, IKeepTempDataResult
 
         public bool Permanent { get; set; }
```

Comment:
If you're feeling particularly industrious, feel free to add doc comments for this property as well 😺